pkg/controller.v1beta1/consts/const.go

```diff
@@ -142,9 +142,9 @@ const (
 	AnnotationIstioSidecarInjectValue = "false"
 
 	// LabelTrialTemplateConfigMapName is the label name for the Trial templates configMap
-	LabelTrialTemplateConfigMapName = "app"
+	LabelTrialTemplateConfigMapName = "katib.kubeflow.org/component"
```

Comment:
Should we use katib.kubeflow.org or automl.kubeflow.org?

Reply:
I am fine with both, it's better to take what is easier for user to see the Katib components.
WDYT @johnugeorge @alculquicondor @kubeflow/wg-training-leads ?

Reply:
Are there other projects that would potentially use `automl.kubeflow.org/component`?

Reply:
Currently, we have only Katib project in Kubeflow for AutoML

Reply:
I think the "component" part might be exclusive to katib.

Of course, you could do `automl.kubeflow.org/component: katib-ui`, but maybe it's overkill.

Reply:
I agree, so in the future if we have more projects for AutoML we can extend it.

Reply:
As we decided on the WG Meeting we will use `katib.kubeflow.org` for now since it is easier to indicate project for the user. 
@gaocegege @alculquicondor @johnugeorge if you are fine with the changes, please leave your `lgtm`

Reply:
lgtm